primitives/src/channel_fixtures.rs

```diff
@@ -67,53 +68,52 @@
 //        .collect()
 //}
 //
-//pub enum ValidatorsOption<'a> {
-//    Pair {
-//        leader: ValidatorDesc,
-//        follower: ValidatorDesc,
-//    },
-//    SpecValidators(SpecValidators),
-//    Generate {
-//        validators_prefix: &'a str,
-//    },
-//}
-//
-//pub fn get_channel_spec(validators_option: ValidatorsOption<'_>) -> ChannelSpec {
-//    use crate::EventSubmission;
-//    use test_util::take_one;
-//
-//    let validators = match validators_option {
-//        ValidatorsOption::Pair { leader, follower } => [leader, follower].into(),
-//        ValidatorsOption::SpecValidators(spec_validators) => spec_validators,
-//        ValidatorsOption::Generate { validators_prefix } => [
-//            get_validator(&format!("{} leader", validators_prefix), None),
-//            get_validator(&format!("{} follower", validators_prefix), None),
-//        ]
-//        .into(),
-//    };
-//
-//    let title_string = Some(<Faker as Lorem>::sentence(3, 4));
-//
-//    let title = take_one(&[&title_string, &None]).to_owned();
-//    let max_per_impression = BigNum::from(<Faker as Number>::between(250, 500));
-//    let min_per_impression = BigNum::from(<Faker as Number>::between(1, 250));
-//    let nonce = BigNum::from(<Faker as Number>::between(100_000_000, 999_999_999));
-//    let min_targeting_score =
-//        take_one(&[&None, &Some(<Faker as Number>::between(1, 500))]).to_owned();
-//
-//    ChannelSpec {
-//        validators,
-//        title,
-//        max_per_impression,
-//        min_per_impression,
-//        targeting: get_targeting_tags(<Faker as Number>::between(0, 5)),
-//        min_targeting_score,
-//        // @TODO: `EventSubmission` fixture issue #27
-//        event_submission: EventSubmission { allow: vec![] },
-//        created: Utc::now(),
-//        active_from: Some(Utc::now()),
-//        nonce,
-//        withdraw_period_start: Utc::now(),
-//        ad_units: Vec::new(),
-//    }
-//}
+pub enum ValidatorsOption<'a> {
+    Pair {
+        leader: ValidatorDesc,
+        follower: ValidatorDesc,
+    },
+    SpecValidators(SpecValidators),
+    Generate {
+        validators_prefix: &'a str,
+    },
+}
+
```

Comment:
There seems to be no need for this. If we want to generate validators for fixtures it would be more constrained around `fees` and not `id` or `url`. In fact, the `URL` & `id` for fixtures should be hardcoded similar to the JS implementation else we would have issues when running tests as the validator id wouldn't be found in the spec.